mysensors/const_24.py

```diff
@@ -0,0 +1,61 @@
+"""MySensors constants for version 2.4 of MySensors."""
+
+import voluptuous as vol
+
+# pylint: disable=unused-import
+from mysensors.const_22 import (  # noqa: F401
+    MAX_NODE_ID,
+    VALID_INTERNAL,
+    VALID_PRESENTATION,
+    VALID_SETREQ,
+    VALID_STREAM,
+    VALID_TYPES,
+    BaseConst,
+    MessageType,
+    Presentation,
+    SetReq,
+    Stream,
+    Internal,
+)
+
+from .handler import HANDLERS_24
+
+
+def get_handler_registry():
+    """Return handler registry for this protocol version."""
+    return HANDLERS_24
+
+
+class SetReq(BaseConst):
+    """MySensors set/req sub-types for protocol 2.4."""
+
+    V_TILT = 58  # Tilt position. Percentage (0-100 %)
```

Comment:
We need to copy all the members from the previous versions here. See eg `const_20.py`.